generators/typescript-v2/ast/src/ast/Type.ts

```diff
@@ -2,12 +2,99 @@ import { assertNever } from "@fern-api/core-utils";
 import { AstNode } from "./core/AstNode";
 import { Writer } from "./core/Writer";
 
-type InternalType = String_;
+type InternalType =
+    | String_
+    | Number_
+    | BigInt_
+    | Boolean_
+    | Array_
+    | Object_
+    | Map_
+    | Tuple
+    | Enum
+    | Any
+    | Unknown
+    | Void
+    | Undefined
+    | Null
+    | Never
+    | Nop;
 
 interface String_ {
     type: "string";
 }
 
+interface Number_ {
+    type: "number";
+}
+
+interface BigInt_ {
+    type: "bigint";
+}
```

Comment:
Is BigInt something we should care about in TS?

Reply:
We unfortunately will need to for some customers.